src/Symfony/Bundle/FrameworkBundle/Tests/Functional/BundlePathsTest.php

```diff
@@ -40,7 +40,7 @@ public function testBundlePublicDir()
     public function testBundleTwigTemplatesDir()
     {
         static::bootKernel(['test_case' => 'BundlePaths']);
-        $twig = static::$container->get('twig');
+        $twig = static::$container->get('twig.alias');
```

Comment:
can't we get private services in tests with the special test container? why is this needed?

Reply:
The `twig` service is still public. We don't want to access to it directly to avoid the deprecation.